conda_build/create_test.py

```diff
@@ -3,44 +3,42 @@
 """
 Module to handle generating test files.
 """
-
+from __future__ import annotations
 
 import json
 import os
-from os.path import exists, join
+from os.path import basename, exists, isfile, join
+from pathlib import Path
 
-from conda_build.utils import copy_into, ensure_list, glob, on_win, rm_rf
+from .metadata import MetaData
+from .utils import copy_into, ensure_list, on_win, rm_rf
 
 
-def create_files(m, test_dir=None):
-    """
-    Create the test files for pkg in the directory given.  The resulting
```

Comment:
Where are the test files created now?

Reply:
not sure what you mean? the docstring was updated and now reads:

```
Copy all test files from recipe over into testing directory.
```

Reply:
Doesn't this call some other functions that create e.g. a Python script that tries to import things the recipe asks for, and then copies them over?

Reply:
No, `create_files` doesn't _create_ any new files, it only copies existing ones. `create_python_files` (and similarly named functions for other supported languages) are where the test scripts you're referencing are created.